docs/source/cql.rst

```diff
@@ -16,17 +21,20 @@ following predicates:
 Formats
 -------
 
-At the moment pygeoapi supports only the CQL dialect with the JSON encoding `CQL-JSON <https://portal.ogc.org/files/96288#simple-cql-JSON>`_.
+At the moment Elasticsearch supports only the CQL dialect with the JSON encoding `CQL-JSON <https://portal.ogc.org/files/96288#simple-cql-JSON>`_.
 
-Providers
----------
-
-As of now the available providers supported for CQL filtering are limited to only :ref:`Elasticsearch <Elasticsearch>`.
+PostgreSQL supports both CQL-JSON and CQL-text dialects, `CQL-JSON <https://portal.ogc.org/files/96288#simple-cql-JSON>`_ & `CQL-TEXT <https://portal.ogc.org/files/96288#simple-cql-text>`_
```

Comment:
`s/&/and/g`

Reply:
Fixed in latest commits by @ximenesuk and @KoalaGeo.